shared/chat/conversation/bottom-banner/index.tsx

```diff
@@ -39,7 +40,11 @@ const InviteBanner = ({users, openSMS, openShareSheet, usernameToContactName}: I
     return (
       <BannerBox color={Styles.globalColors.blue} gap="xtiny">
         <BannerText>Last step: summon {theirName}!</BannerText>
-        <Button label="Send install link" onClick={() => openSMS(users[0].slice(0, -6))} mode="Secondary" />
+        <Button
+          label={Flags.wonderland ? '🐇 Send install link' : 'Send install link'}
```

Comment:
there are three of these buttons in this file. you've only got the one that's mobile and one-on-one. are we doing this for desktop also? assuming no, then probably just need one more. down a few lines.

Reply:
good catch. Afaik the 🐇 task includes 🐇 on desktop. Will get in future PR